theano/gpuarray/opt.py

```diff
@@ -135,9 +180,10 @@ def local_opt(node):
                         context_name = i.owner.inputs[0].type.context_name
                         replace = True
                         break
+                clients = [c for o in node.outputs for c in o.clients]
```

Comment:
Move it back inside the if. It don't look as being used when the if isn't executed.